web/styles/settings.css

```diff
@@ -656,11 +652,46 @@ input[type="checkbox"] {
     margin: 10px 0;
 }
 
-.add-new-default-stream-box {
-    & input[type="text"] {
-        padding: 6px;
+#add-default-stream-modal {
+    .dropdown-list-widget {
+        display: inline;
+    }
+
+    .dropdown-widget-button {
+        display: inline-flex;
+    }
+
+    .dropdown-menu {
```

Comment:
Same here for `dropdown-menu` class, is it used here?

Reply:
No, it was used in the old dropdown widget. Removed. Thanks.